Simbody/src/SimbodyMatterSubsystemRep.cpp

```diff
@@ -5400,6 +5401,56 @@ void SimbodyMatterSubsystemRep::multiplyByMInv(const State& s,
 }
 //............................. CALC M INVERSE F ...............................
 
+//==============================================================================
+//                            EU MULTIPLY BY SQRT M INV
+//==============================================================================
+// Calculate udot = sqrt(M^-1) v. We also get spatial accelerations A_GB for
```

Comment:
Why udot here rather than u? And the spatial acceleration comment seems like a copy-and-paste error. Please revise.